src/types/index.ts

```diff
@@ -98,7 +98,7 @@ export type StepWithData = StepBase & {
 export type QuestionState = {
   available_points: AvailablePoints;
   is_completed: boolean;
-  answer_id?: ID;
+  answer_id: number;
```

Comment:
I'm thinking we want this to not be optional and use 0 to mean no-value from the assessments side, since the api client expects a value, but I might be thinking about this wrong.

Reply:
it should be optional on input but required in the callback payload